kata-webhook/webhook-check.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# Copyright (c) 2021 Red Hat
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Run this script to check the webhook is deployed and working
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+webhook_dir=$(dirname $0)
+source "${webhook_dir}/../lib/common.bash"
+source "${webhook_dir}/common.bash"
+
+readonly hello_pod="hello-kata-webhook"
+# The Pod RuntimeClassName for Kata Containers.
+RUNTIME_CLASS="${RUNTIME_CLASS:-"kata"}"
+
+cleanup() {
+	{
+	kubectl get -n ${WEBHOOK_NS} pod/${hello_pod} && \
+		kubectl delete -n ${WEBHOOK_NS} pod/${hello_pod}
+	} &>/dev/null
+}
+trap cleanup EXIT
+
+# Check the deployment exists and is available.
+#
+check_deployed() {
+	kubectl get -n ${WEBHOOK_NS} deployment/${WEBHOOK_SVC} &>/dev/null || \
+		die "The ${WEBHOOK_SVC} deployment does not exist"
+
+	kubectl wait -n ${WEBHOOK_NS} deployment/${WEBHOOK_SVC} \
+		--for condition=Available --timeout 60s &>/dev/null || \
+		die "The ${WEBHOOK_SVC} deployment is unavailable after 60s waiting"
+}
+
+# Check the webhook is working as expected.
+#
+check_working() {
+	kubectl get -n ${WEBHOOK_NS} pod/${hello_pod} &>/dev/null && \
+		die "${hello_pod} pod exists, cannot reliably check the webhook"
+
+	cat <<-EOF | kubectl apply -f -
+	kind: Pod
+	apiVersion: v1
+	metadata:
+	  name: ${hello_pod}
+	  namespace: ${WEBHOOK_NS}
+	spec:
+	  restartPolicy: Never
+	  containers:
+	    - name: ${hello_pod}
+	      image: quay.io/prometheus/busybox:latest
```

Comment:
Not absolutely required, but it might be worth making this into a variable as we have this URL (and other busybox image URL variants) scattered all over the place ;)

Reply:
@jodh-intel I can turn it into a variable, yes. But maybe a more complete solution would be:

* Add a busybox entry on `versions.yaml` (possible under `container_images` section)
* Move `get_test_version()` (and dependencies methods) from `.ci/lib.sh` to `lib/common.bash`
* Finally adjust `kata-webhook/webhook-check.sh` to use `get_test_version`

Worth doing that second solution?